css/icons.css

```diff
@@ -116,10 +116,20 @@
 }
 
 /* To be used in new conversation / invitation handler dialogs */
+.talk-modal .avatar.icon.icon-user.icon--dark {
+	background-image: url('../img/icon-conversation-user-dark.svg')
+}
+
+.talk-modal .avatar.icon.icon-user.icon--bright {
+	background-image: url('../img/icon-conversation-user-bright.svg')
+}
+
+.talk-modal .avatar.icon.icon-contacts.icon--dark,
 .conversation-icon .icon-conversation-group.icon--dark {
 	background-image: url('../img/icon-conversation-group-dark.svg')
 }
 
+.talk-modal .avatar.icon.icon-contacts.icon--bright,
 .conversation-icon .icon-conversation-group.icon--bright {
 	background-image: url('../img/icon-conversation-group-bright.svg')
 }
```

Comment:
Why don't you omit talk-modal class?  It will be applied on all avatar wrappers at once without the need to specify a parent class, unless there are some avatars that need to stay with the old background ?

Reply:
Maybe we should reconsider how the whole style list from that class works)

Reply:
Yes, we take the avatar style based on itself only regardless of its parent or where it was rendered.